Source/carthage/Archive.swift

```diff
@@ -61,10 +94,11 @@ public struct ArchiveCommand: CommandType {
 			.collect()
 			.flatMap(.Merge) { paths -> SignalProducer<(), CarthageError> in
 				if paths.isEmpty {
```

Comment:
This only errors if _none_ of the frameworks are found. We should error if any of the frameworks is missing.